### DISCUSSION TITLE
Followed a guide to upgrade to Python 3.10 on Ubuntu and it bricked my PC

### DISCUSSION THREAD

**Discussion by codewriter3000** [CloudBytes-Academy/CloudBytes.dev#93] [Q&A]
I was on Ubuntu 20.04 and I followed this guide word for word on how to upgrade from Python 3.8 to Python 3.10 and my PC is bricked and I have to reinstall Ubuntu all over again: https://cloudbytes.dev/snippets/upgrade-python-to-latest-version-on-ubuntu-linux

**Comment by rehanhaider** [comment]
> I followed this guide word for word on how to upgrade from Python 3.8 to Python 3.10 and my PC is bricked and I have to reinstall Ubuntu all over again: https://cloudbytes.dev/snippets/upgrade-python-to-latest-version-on-ubuntu-linux

Can you please provide more details? Steps, screenshots, errors messages, etc.

**Comment by codewriter3000** [comment]
![image](https://user-images.githubusercontent.com/58712923/152089342-3cecd65d-b3a7-495f-b9ea-b7214d1d6cf9.png)
This is the part that bricks Ubuntu. I learned the hard way that there is software in the Ubuntu operating system that depends on Python 3.8.

**Comment by rehanhaider** [comment]
Can you please also provide the exact version of Ubuntu you are using?

**Reply by codewriter3000** [reply]
Ubuntu 20.04

**Reply by codewriter3000** [reply]
It would be in your best interest to edit the article before anybody else bricks their Ubuntu.

**Reply by rehanhaider** [reply]
Not untill I am able to reproduce the issue. So far I have not been able to.

**Reply by ejfisc** [reply]
I'm on Zorin OS 16 which is based on Ubuntu 20.04.2 and I just experienced the same issue, following your guide exactly. I highly suggest you modify or remove the article before anybody else bricks their OS.

**Reply by Carlos-Mareco** [reply]
Please remove this article, I just followed the steps in the article and important parts of Linux Mint 20.3 Cinnamon has stopped working and I can't start my OS.

**Reply by vincecorradetti** [reply]
I think you should remove the article!

**Comment by K2AAE** [comment]
I found that Nvidia Jetson LT4 Ubuntu 18.04 appeared to work, got 3.10 installed and ran some scripts but when I rebooted the system failed, from the console the /var/log/syslog was full of apt errors as in python3.6-apt
No big, they had released a new LT4 (4.6.1) so I installed it and got python 3.6, again. Putting my Jetson Nanos back in the Pet Rock farm, after all this time they still have not figured out how to boot from a thumb drive or move on from 5 year old software.. USB 3.0 is so much faster to boot from.
